.github/workflows/tests.yml

```diff
@@ -0,0 +1,185 @@
+        working-directory: packages/cli
+      - run: npm run coverage --prefix=packages/cli
+
+  webdriverjs:
+    strategy:
+      matrix:
+        node: [16, 18, 20]
+    runs-on: ubuntu-latest
+    needs: lint
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+      - uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
+      # HACK: Force a TTY to enable browser-driver-manager to manipulate stdout.
+      - shell: 'script -q -e -c "bash {0}"'
+        run: npx browser-driver-manager install chrome chromedriver --verbose
+      - run: npm run build --prefix=packages/webdriverjs
+      - run: npm run coverage --prefix=packages/webdriverjs
+      - run: npm run test:esm --prefix=packages/webdriverjs
+
+  webdriverio:
+    strategy:
+      matrix:
+        node: [16, 18, 20]
+    runs-on: ubuntu-latest
+    needs: lint
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+      - uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
+      # HACK: Force a TTY to enable browser-driver-manager to manipulate stdout.
+      - shell: 'script -q -e -c "bash {0}"'
+        run: npx browser-driver-manager install chrome chromedriver --verbose
+      - run: npm run build --prefix=packages/webdriverio
+      - run: npm run coverage --prefix=packages/webdriverio
+      - run: npm run test:esm --prefix=packages/webdriverio
+
+  reporter_earl:
+    strategy:
+      matrix:
+        node: [16, 18, 20]
+    runs-on: ubuntu-latest
+    needs: lint
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+      - uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
+      - run: npm run build --prefix=packages/reporter-earl
+      - run: npm run test --prefix=packages/reporter-earl
+      - run: npm run test:esm --prefix=packages/reporter-earl
+
+  react:
+    strategy:
+      matrix:
+        node: [16, 18, 20]
+    runs-on: ubuntu-latest
+    needs: lint
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+      - uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
+      - run: npm run build --prefix=packages/react
+      - run: npm run test --prefix=packages/react
+      - run: npm run test:esm --prefix=packages/react
+
+  playwright:
+    strategy:
+      matrix:
+        node: [16, 18, 20]
```

Comment:
How is this working when we said we had to [force pin to node 14](https://github.com/dequelabs/axe-core-npm/blob/ca07660635fbc6c0e498d4326213b3901bdf3698/.circleci/config.yml#L207) due to WDIO sync?

Reply:
We don't support WDIO sync. See https://github.com/dequelabs/axe-core-npm/pull/708

Reply:
Oh right, forgot we finally resolved that issue.